packages/block-editor/src/components/default-block-appender/index.js

```diff
@@ -53,14 +53,16 @@ export function DefaultBlockAppender( {
 			data-root-client-id={ rootClientId || '' }
 			className="wp-block block-editor-default-block-appender"
 		>
-			<TextareaAutosize
-				role="button"
-				aria-label={ __( 'Add block' ) }
-				className="block-editor-default-block-appender__content"
-				readOnly
-				onFocus={ onAppend }
-				value={ showPrompt ? value : '' }
-			/>
+			<p>
```

Comment:
Why not just make this content editable and add the focus handler here?

Reply:
With content editable, it's guaranteed to be the same, no? We don't have to deal with the annoying text area.